commands.js

```diff
@@ -526,6 +526,7 @@ exports.commands = {
 				this.addModCommand("" + user.name + " turned on modjoin.");
 			} else if (target in Config.groups) {
 				if (room.battle && !this.can('makeroom')) return;
+				if (room.isPersonal && !user.can('makeroom') && target !== '+') return this.errorReply("/modjoin - Access denied from setting modjoin past + Personal Rooms.");
```

Comment:
There's a difference between names used in the code and names used to talk to users.

`isPersonal` is only what they're called in the code. In reality, we call them "group chats", not "Personal Rooms".

Also, "past + group chats" is grammatically weird. Try "past + in group chats".